main.tf

```diff
@@ -60,7 +68,7 @@ resource "hcp_vault_cluster" "main" {
   }
 
   lifecycle {
-    # see https://www.terraform.io/language/meta-arguments/lifecycle#prevent_destroy
+    # see https://developer.hashicorp.com/terraform/language/meta-arguments/lifecycle#prevent_destroy
     prevent_destroy = true
```

Comment:
Maybe expose this via a variable (default to true), to allow destruction using TF from the root module?

Reply:
README calls out removing from state, but that will not clean up the resource in HCP.

Reply:
Sadly, `prevent_destroy` isn't something that's eligible for variables right now, so I've opted to hard-code this.

I consider the cost of removing a Cluster manually (vs. fully automated and accidentally through `terraform destroy`) is negligible, compared to the cost of re-creating a wiped-out cluster (either through contacting `support@` or by re-deploying it and vending new tokens).

So, while _yes_ you are absolutely right that it will not clear up the resource in HCP, this is intentional and not something I want to "fix" at this point :heart: 

The note in the README is to ensure users know how to remove the cluster _if_ they need to!

Reply:
I've gone ahead and made the `manual` part more clear in the docs